src/extra/buttons.css

```diff
@@ -6,42 +6,42 @@
   --_accent: initial /* your color */;
   --_text: initial /* your text color */;
   --_size: initial /* your size */;
-  
```

Comment:
some linter is making this PR look like a much bigger change than it needs to be! can you commit only the lines that need to change please 🙂

Reply:
Huh. I didn't see those changes...Weird. I'll reset HEAD^ and git add -p...